src/tests/GC/Stress/Framework/RFLogging.cs

```
@@ -120,7 +120,6 @@ private void LogWorker()
                 }
                 catch (IOException e)
                 {
-                    ReliabilityFramework.MyDebugBreak(String.Format("LogWorker IOException:{0}", e.ToString()));
```

We probably want to throw an exception here still after logging?

---

Same as above: if debugger is present => break else, log and throw an exception.